src/include/homestore/blkdata_service.hpp

```diff
@@ -131,6 +132,11 @@ class BlkDataService {
      */
     BlkReadTracker* read_blk_tracker() { return m_blk_read_tracker.get(); }
 
+    /**
+     * @brief : start data service;
+     */
+    void start();
```

Comment:
Is it intentionally meant for something like HomeObject to call this? If not, let's hide it somewhere internally yeah?

Reply:
right, it can be a private member function, will move it with next PR.